docs/NMOS With JPEG XS.md

```diff
@@ -67,10 +67,13 @@ An example Flow resource is provided in the [Examples](../examples/).
 
 ### Senders
 
-The Sender resource MUST indicate `urn:x-nmos:transport:rtp` or one of its subclassifications for the `transport` attribute.
+For Nodes transmitting JPEG XS using the RTP payload mapping defined by RFC 9134, the Sender resource MUST indicate `urn:x-nmos:transport:rtp` or one of its subclassifications for the `transport` attribute.
 Sender resources provide no indication of media type or format, since this is described by the associated Flow resource.
 
 The SDP file at the `manifest_href` MUST comply with the requirements of RFC 9134.
+Additionally, the format-specific parameters defined by RFC 9134 are made REQUIRED in the same circumstances that a corresponding Flow attribute MUST be included, in order that the information provided to an NMOS Controller through a Flow is also provided to a Receiver through the SDP file.
```

Comment:
Conformance language lawyer needed... I'm not sure if the "Flow attribute MUST be included" would be better as just "Flow attribute is included", since this is talking _about_ a requirement expressed above rather than expressing that requirement?

Reply:
What about this ... but I'm not a layer :)

"Additionally, the format-specific parameters defined by RFC 9134 corresponding to REQUIRED Flow attributes, as per the IS-04 JSON schemas, are made REQUIRED and MUST be included, in order that the information provided to an NMOS Controller through a Flow is also provided to a Receiver through the SDP file."

Reply:
Two small objections to that.

First, that default values for some properties in the schemas mean that properties like `interlace_mode` that aren't listed as `"required"` effectively always communicate information even when omitted in the Flow resource, along with the similar scenario in SDP for some optional parameters like `interlace`.

Second that we want to apply this also to the attributes defined in this spec, of course. 🙂

This may all be irrelevant if we consider John's comments in the other thread on this PR and decide that the benefit of doing this at all is outweighed by the costs.

Reply:
👍

So if you're relying on the default value in the NMOS Flow, you may still require an explicit value in the SDP, which I tried and failed to capture in my wording too. It's tricksy!

Reply:
What about this new flavor. An attempt to focus on providing same REQUIRED information to both Controller and Receiver. After this normative paragraph we can provide non-normative examples of what it means for JPEG-XS.

"Additionally, in order to provide to a Receiver, through SDP file format-specific parameters, the information provided to an NMOS Controller through REQUIRED Flow attributes, format-specific parameters defined by RFC 9134 are made REQUIRED in the same circumstances that corresponding Flow attributes are made REQUIRED by IS-04 JSON schemas or this specification."

Reply:
```suggestion
Additionally, the SDP file MUST convey, so far as the defined parameters allow, the same information about the stream as conveyed by the Source, Flow and Sender attributes.
```